core/test.py

```diff
@@ -1,8 +1,37 @@
+from abc import ABCMeta
+
 from unittest import TestCase
 
+from core import app
+from user.models import User
+from blog.models import Article
 from .utils import clean_html
 
 
+class BaseTest(TestCase):
+    __metaclass__ = ABCMeta
```

Comment:
funny how we discussed we never need to use metaclass.
What's the point of using metaclass here? 
If you remove this it will still work.